config/exchanges/AT_CZ.yaml

```
@@ -2,6 +2,6 @@ lonlat:
   - 15.486554
   - 48.909846
 parsers:
-  exchange: ENTSOE.fetch_exchange
-  exchangeForecast: ENTSOE.fetch_exchange_forecast
+  exchange: CZ.fetch_exchange
+  exchangeForecast: CZ.fetch_exchange_forecast
```

This function has not been implemented in the CZ parser. Let's revert it to ENTSOE.
```suggestion
  exchangeForecast: ENTSOE.fetch_exchange_forecast
```